blazor/diagram/getting-started/deploy-maui-windows.md

```diff
@@ -0,0 +1,259 @@
+---
+layout: post
+title: Deploy Diagram Component in Blazor MAUI in windows | Syncfusion
+description: Learn how to getting started with Diagram control in Blazor MAUI application
+platform: Blazor
+control: Diagram Component
+documentation: ug
+---
+
+# Getting started with Diagram Component in the Blazor MAUI app.
+
+In this section, we'll guide you through the process of adding Syncfusion's Blazor Diagram Component to your Blazor Maui app. We'll break it down into simple steps to make it easy to follow.
+
+## Prerequisites
+
+To use the MAUI project templates, install the Mobile development with the .NET extension for Visual Studio. For more details, refer to [here](https://learn.microsoft.com/en-us/dotnet/MAUI/get-started/installation?tabs=vswin).
+
+## Create a new Blazor MAUI App in Visual Studio
+
+Create a new Blazor MAUI app and name it **DiagramComponentGettingStarted**.
+
+
+## Install Diagram NuGet package in Blazor Maui App
+
+Add the following NuGet packages into the Blazor Maui app.
+
+* [Syncfusion.Blazor.Diagram](https://www.nuget.org/packages/Syncfusion.Blazor.Diagram) 
+* [Syncfusion.Blazor.Themes](https://www.nuget.org/packages/Syncfusion.Blazor.Themes)
+
+## Register Syncfusion Blazor Service
+
+* In the **~/_Imports.razor** file, add the following namespaces:
+
+{% tabs %}
+{% highlight razor tabtitle="~/_Imports.razor" %}
+
+@using Syncfusion.Blazor 
+@using Syncfusion.Blazor.Diagram
+
+{% endhighlight %}
+{% endtabs %}
+
+* Register the Syncfusion Blazor Service in the **~/MauiProgram.cs** file.
+
+{% tabs %}
+{% highlight c# tabtitle="~/MauiProgram.cs" hl_lines="3 20 28" %}
+
+using Microsoft.Extensions.Logging;
+using MauiBlazorWindow.Data;
+using Syncfusion.Blazor;
+
+namespace MauiBlazorWindow;
+
+public static class MauiProgram
+{
+    public static MauiApp CreateMauiApp()
+    {
+        var builder = MauiApp.CreateBuilder();
+        builder
+            .UseMauiApp<App>()
+            .ConfigureFonts(fonts =>
+            {
+                fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+            });
+
+        builder.Services.AddMauiBlazorWebView();
+        builder.Services.AddMemoryCache();
+
+#if DEBUG
+        builder.Services.AddBlazorWebViewDeveloperTools();
+        builder.Logging.AddDebug();
+#endif
+
+        builder.Services.AddSingleton<WeatherForecastService>();
+        builder.Services.AddSyncfusionBlazor();
+        return builder.Build();
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Adding stylesheet and script
+
+Add the following stylesheet and script to the head section of the **~/wwwroot/index.html** file.
+
+If you are using Syncfusion.Blazor && Syncfusion.Blazor.Themes nuget package in your application you can refer below script.
```

Comment:
If you are using Syncfusion.Blazor && Syncfusion.Blazor.Themes NuGet package in your application, refer to the following script.